### PR TITLE
Move zeroes

### PR DESCRIPTION
**Pull request by RanaSalameh20**

### CODE REVIEW

File: Move-Zeros/Move.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Move_Zeros
+{
+    public class Move
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs]: again, class name is always noun not Verb

File: Move-Zeros/Program.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Move_Zeros
+{
+    class Program : Move
```
Comment (hayatzsala) [inline Move-Zeros/Program.cs]: Why Program class inherits Move class !??

File: Move-Zeros/Move.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+        }
+         void Swap(int[] array, int i, int j)
+        {
+            int temp = array[i];
+            array[i] = array[j];
+            array[j] = temp;
+        }
+
+        internal void Print(int[] numbers)
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs]: let's be more consistent, either all methods and class public or internal

File: Move-Zeros/Move.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            foreach (int number in numbers)
+            {
+                Console.Write(number + ",");
+
+            }
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs]: Read about string.join and try to use it here instead of the loop

File: Move-Zeros/Move.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                }
+            }
+
+        }
+         void Swap(int[] array, int i, int j)
+        {
+            int temp = array[i];
+            array[i] = array[j];
+            array[j] = temp;
+        }
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs]: please add an access modifier for this method

File: Move-Zeros/Move.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        public void MoveZeroesToRight(int[] numbers)
+        {
+            int j = 0;
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                if (numbers[i] != 0)
+                {
+                    Swap(numbers, i, j); // Partitioning the array
+                    j++;
+                }
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs:16]: actually, I don't see the order of non-zeros numbers.

File: Move-Zeros/Move.cs [hunk 7]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Move_Zeros
```
Comment (hayatzsala) [inline Move-Zeros/Move.cs]: NOT recommended to use special characters in namespace ,project, solution name

**Review by hayatzsala — CHANGES_REQUESTED** [review]

File: Move-Zeros/Program.cs (hunk 2 above)
Comment (RanaSalameh20) [reply Move-Zeros/Program.cs]: by mistake

**Pull request merged**